tools/sdk-cli/README.md

```diff
@@ -3,8 +3,9 @@
 A simple cli tool that sends commands to the sdk.
 
 ## Run
-```cargo run```
+```cargo run -- --data_dir <data directory>```
```

Comment:
When trying a custom `data_dir`, the CLI still uses the default `history.txt`.

For example:

```
mkdir test-dir
cargo run -- --data_dir test-dir
```

Pressing up/down arrows after it starts cycles through the `history.txt` from the default location.